statsdaemon.go

```diff
@@ -289,12 +289,12 @@ func processGauges(buffer *bytes.Buffer, now int64) int64 {
 
 		switch {
 		case hasChanged:
-			fmt.Fprintf(buffer, "%s %d %d\n", bucket, currentValue, now)
+			fmt.Fprintf(buffer, "%s %.0f %d\n", bucket, currentValue, now)
```

Comment:
I'd vote for not truncating and accepting the increased verbosity. Feels like it will be cleaner not having to have a truncate case.